<launch>
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />
  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyzrgb nodelet_manager">
    <remap from="rgb/camera_info" to="/camera/color/camera_info"/>
    <remap from="depth_registered/image_rect" to="/camera/aligned_depth_to_color/image_raw"/>
    <remap from="rgb/image_rect_color" to="/camera/color/image_raw"/>
  </node>

</launch>
